lib/displayObject/container/mask/sprite/particle/Particle.js

```diff
@@ -562,6 +567,42 @@ class Emitter
 		return this._out;
 	}
 
+	/**
+	 * textures
+	 * @getter
+	 * This function is a getter for the member textures.
+	 * @return {Array}  textures  An array of textures to use for the particles. 
+	 */
+	get textures()
+	{
+		return this._textures;
+	}
+
+	/**
+	 * textures
+	 * @setter
+	 * This function is a setter for the member textures.
+	 * @param {Array}  textures  An array of textures to use for the particles.
+	 */
+	set textures(textures)
+	{
+		if (!Array.isArray(textures))
+				throw new TypeError("textures must be an array.");

```

Comment:
Don't know if this is a Github displaying bug, but it seems there is an issue with the indents here, no ?